.changeset/neat-hotels-thank.md

```diff
@@ -0,0 +1,5 @@
+---
+'@aws-amplify/backend-storage': patch
```

Comment:
nit: technically this should be a minor

Reply:
Got it, I was following the similar PR for functions but that should have been a minor too.